### PR TITLE
W-13975728 add flags to api manager cli kl

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (glenn-rodgers-sf) [inline .vscode/settings.json:1]: Please delete this file.

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 2]
```diff
@@ -495,7 +495,7 @@ Undeprecates the API instance passed in `<apiInstanceId>`.
 
 This command accepts only the default `--help`, general, and authentication flags.
 
-[[api-mgr-contract-delete]]
+[[api-mgr-contract-delete]]:
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc:498]:
```suggestion
[[api-mgr-contract-delete]]
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 3]
```diff
@@ -523,11 +523,10 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 |===
 |Flag |Description |Example
-|`--limit` | Number of results to retrieve | `--limit 2`
-|`--offset` | Offsets the amount of APIs passed | `--offset 3 643404`
+|`--limit <value>` | Number of results to retrieve | `--limit 2`
+|`--offset <value>` | Offsets the amount of APIs passed | `--offset 3`
 | `--output <value>` | Specifies the response format, supported values are `table` (default) and `json`.|  `--output json`
-|`--sort` | Sorts the results in the field name passed | `--sort "Latest Version" 643404`
-|===
+|`--sort <value>` | Sorts the contracts based on the supported values | `--sort id` |===
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc:529]: This is where you should add the info provided by Franco.

**Review by fbertizzolo-mulesoft — COMMENTED** [review]
LGTM

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 4]
```diff
@@ -523,11 +523,10 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 |===
 |Flag |Description |Example
-|`--limit` | Number of results to retrieve | `--limit 2`
-|`--offset` | Offsets the amount of APIs passed | `--offset 3 643404`
+|`--limit <value>` | Number of results to retrieve | `--limit 2`
+|`--offset <value>` | Offsets the amount of APIs passed | `--offset 3`
 | `--output <value>` | Specifies the response format, supported values are `table` (default) and `json`.|  `--output json`
-|`--sort` | Sorts the results in the field name passed | `--sort "Latest Version" 643404`
-|===
+|`--sort <value>` | Sorts the contracts by the criteria associated with their client applications, supported values are `createdDate`, `id`, and `name`. | `--sort id` |===
```
Comment (sathya0) [inline modules/ROOT/pages/_partials/api-mgr.adoc:529]: I might be wrong but is the formatting after line 529 messed up?

**Review by sathya0 — APPROVED** [review]

**Pull request merged**